CSS/styles.css

```diff
@@ -0,0 +1,1015 @@
+/* #region base */
+
+/* From min-width, I find the answer on how to set the min-width of the page here
+https://stackoverflow.com/questions/3802455/min-width-in-window-resizing
+That's why I use important */
+body {
+  color: gray;
+  font-size: 13px;
+  line-height: 18px;
+  font-family: Arial, Helvetica, sans-serif;
+  overflow-x: hidden;
+  min-width: 375px;
+  width: auto !important;
+  width: 375px;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+a:hover {
+  text-decoration: none;
+  color: inherit;
+}
+
+ul li {
+  list-style: none;
+}
+
+header {
+  background-color: red;
+  color: white;
+  font-weight: bold;
+  position: relative;
+  height: 50px;
+  width: 100vw;
+}
+
+/* #endregion base */
+
+/* #region layout */
+
+/* #region header */
+
+/* #region header top */
+.page-header__date {
+  height: 20px;
+  width: 100%;
+}
+
+.header__item {
+  height: 50px;
+}
+
+.page-header__links {
+  height: 30px;
+  width: 100%;
+}
+
+.page-header__link-item {
+  display: inline-block;
+  text-transform: uppercase;
+  font: bold 14px/14px "Roboto Condensed", sans-serif;
+  padding: 6px 3px;
+}
+
+.page-header__menu-icon {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 50px;
+  height: 50px;
+}
```

Comment:
#### Implementation details

* [ ] All your layout should use Bootstrap CSS classes **everywhere** for styling (for example [Grid layout](https://getbootstrap.com/docs/4.3/layout/overview/) and [Flex](https://getbootstrap.com/docs/4.3/layout/utilities-for-layout/#flexbox-options)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/using_bootstrap/bootstrap_classes_requirement.png) from [Pathwrigth](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540308/))

Reply:
Thanks for the review! :) I change the class and use bootstrap to display flex :D